app/views/comment_mailer/notify.html.erb

```diff
@@ -14,6 +14,13 @@ Hi! There's been a response to a discussion you're involved in. Do NOT reply to
 
 <hr />
 
+<p>Look like spam?
+  <% if @user.can_moderate? %>
```

Comment:
So here's what I'm worried about, are we notifying in a bulk BCC, or do we actually have unique user record each time it's sent? 

I can't find where this is used to see what the `@user` variable is, but let's check!

https://github.com/publiclab/plots2/search?q=CommentMailer.notify&unscoped_q=CommentMailer.notify

Reply:
Each time a new comment is created this `CommentMailer.notify`  send emails to users subscribed to the tags or node. So, I have added these buttons only on normal notification emails.
Here's the effective mailer https://github.com/publiclab/plots2/blob/master/app/mailers/comment_mailer.rb#L7-L12

Reply:
@jywarren

Reply:
yeah, that's great! I just want to know where `user` is specified -- like, do you know where this `.notify()` is called from, so we can be sure it represents the recipient user instead of the comment author? I think we may want to rename the parameter a little bit to be sure of it, since `user` is ambiguous, you know?

Reply:
here it is @jywarren 
https://github.com/publiclab/plots2/blob/master/app/models/comment.rb#L124-L130